assets/js/atomic/blocks/product-elements/sale-badge/types.ts

```diff
@@ -1,4 +1,4 @@
 export interface BlockAttributes {
-	productId: number;
-	align: 'left' | 'center' | 'right';
+	productId?: number;
+	align?: 'left' | 'center' | 'right';
```

Comment:
This part had been changed to prevent a TS error in the following section:
- https://github.com/woocommerce/woocommerce-blocks/blob/04f3f4f2bdddae3babf0bab092f58488d109bedd/assets/js/blocks/cart/cart-cross-sells-product-list/cart-cross-sells-product.tsx#L51